cookbook/templating/namespaced_paths.rst

```diff
@@ -74,6 +74,10 @@ directory:
             );
         ));
 
+.. caution::
+
+    These paths are not warmed during cache warming. This is fixed in 2.8.
```

Comment:
What about: "Prior to 2.8, templates in custom namespaces are not pre-compiled by Symfony's cache warmup process. They are compiled on demand. This may cause problems if two simultaneous requests are trying to use the template for the first time."